gnmi_dump/gnmi_dump.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"github.com/sonic-net/sonic-gnmi/common_utils"
+)
+
+func main() {
```

Comment:
>main [](http://example.com/codeflow?start=5&length=4)

If this is a new CLI, add `usage()`. Normally we require file level and function level comments, usage is also kind of comments.